apps/prairielearn/src/lib/client/page-context.ts

```diff
@@ -9,8 +11,9 @@ import {
 
 const RawPageContextSchema = z.object({
   authz_data: z.object({
-    has_course_instance_permission_edit: z.boolean(),
-    has_course_instance_permission_view: z.boolean(),
+    // TODO: Type these more accurately into a course instance version.
+    has_course_instance_permission_edit: z.boolean().optional(),
```

Comment:
This was an outstanding bug in the page context schema, it assumed a course instance page.